sagemaker/launch/launch_utils.py

```diff
@@ -0,0 +1,91 @@
+"""
+    Copyright 2025 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Utility functions for inference
+"""
+
+import os
+import shutil
+import tarfile
+
+
+def wrap_model_artifacts(path_to_model, path_to_yaml, path_to_json, path_to_entry,
+                         output_tarfile_name='model', output_path=None):
+    """ A utility function to zip model artifacts into a tar package
+
+    According to SageMaker's specification of the `Model Directory Structure
+    https://sagemaker.readthedocs.io/en/stable/frameworks/pytorch/using_pytorch.html#model-directory-structure`_,
+    this function will put the entry point file into a sub-folder,named `code`, and then zip the
+    `model.bin`, the `**.yaml`, the `**.json` file, and the `code` sub-folder into a tar package
+    with the name specified by the wrap_name argument, and save it to the `output_path`.
+
+    This function will check if given files exist. Non-existance will raise an error.
+    
+    Parameters
+    ----------
+    path_to_model: str
+        The path of GraphStorm's `model.bin` file.
+    path_to_yaml: str
+        The path of the YAML file generated by GraphStorm during model training.
+    path_to_json: str
+        The path of JSON file generated by GraphStorm gconstruct or GSProcessing.
+    path_to_entry: str
+        The path of the entry point file for a specific task. The file will be put into a
+        sub-folder, named 'code'.
+    output_package_name: str
+        The name of the tar package. Default is `model`.
+    output_path: str
+        The folder where the output tar package will be saved. If not provided, will
+        raise an error. Default is None, meaning not provided.
```

Comment:
If we want to raise an error when the output is not provided, why not make the value not have a default, so that it's required to call the function?

Reply:
remove the default and revise test cases.